modules/stratos/Makefile

```diff
@@ -20,5 +20,9 @@ install:
 upgrade:
 	./upgrade.sh
 
+.PHONY: reachable
+reachable:
+	./reachable.sh
+
 .PHONY: all
-all: clean chart gen-config install
+all: clean chart gen-config install reachable
```

Comment:
I would prefer to not add `reachable` here, for those clusters were you can't access the DNS domain outside of the cluster for security reasons (such as the ones in kubecf pipeline).